modules/highgui/include/opencv2/highgui.hpp

```diff
@@ -364,6 +364,24 @@ If there are several HighGUI windows, any of them can be active.
  */
 CV_EXPORTS_W int waitKey(int delay = 0);
 
+/** @brief Polls for a pressed key.
+
+The function pollKey polls for a key event without waiting. It returns the code of the pressed key
+or -1 if no key was pressed since the last invocation.
+
+@note
+
+This function serves a similar function as waitKey. Either function needs to be
+called periodically for normal event processing unless HighGUI is used within an environment that
+takes care of event processing.
```

Comment:
This note is [malformed](http://pullrequest.opencv.org/buildbot/export/pr/19411/docs/d7/dfc/group__highgui.html#ga6d20fbd3100ec3badc1eaa653aff99d7).

Need to remove new lines.

Reply:
I don't quite understand. remove Line 373 which is a blank line between `@note` and its body? or reformat paragraph to use 72-80 characters of width consistently?